### ISSUE TITLE
Ghost version >= 3.14.0 causes an issue

### ISSUE THREAD

**Issue by l50** [l50/techvomit#2]
Found that 3.13.4 is the last working version:
```
image: bitnami/ghost:3.13.4
```

**Comment by l50** [comment]
I see this when I run `docker logs nginx-proxy` with a version >= 3.14.0:
```
*1 connect() failed (111: Connection refused) while connecting to upstream, client: x.x.x.x, server: mysystem.com, request: "GET /favicon.ico HTTP/2.0"
```

**Comment by l50** [comment]
Fixed via version pinning to `3.13.4`.
